lib/retirement_swap/subject.rb

```diff
@@ -20,8 +20,16 @@ def initialize(id, attributes)
       end
     end
 
+    def test?
+      category == 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.